### PR TITLE
Update libpcap to 1.5.3 version

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

File: rpm/libpcap.spec [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+make DESTDIR=$RPM_BUILD_ROOT install
+rm -f $RPM_BUILD_ROOT%{_libdir}/libpcap.a
+
+%post -p /sbin/ldconfig
+
+%postun -p /sbin/ldconfig
+
+%files
+%defattr(-,root,root)
+%doc LICENSE README CHANGES CREDITS
```
Comment (saukko) [inline rpm/libpcap.spec]: Leave the license and move rest to -devel I would say.

File: rpm/libpcap.spec [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+%{_libdir}/libpcap.so.*
+%{_mandir}/man7/pcap*.7*
```
Comment (saukko) [inline rpm/libpcap.spec]: Man page also could go to devel (or actually -docs or something but not sure if we have proper policy for that for now), we don't need it on device.

File: rpm/libpcap.spec [hunk 3]
```diff
@@ -0,0 +1,77 @@
+Name: libpcap
+Version: 1.5.3
+Release: 1
+Summary: A system-independent interface for user-level packet capture
+Group: Development/Libraries
+License: BSD with advertising
```
Comment (saukko) [inline rpm/libpcap.spec]: I wonder what this advertising here is, the license file to me seems generic BSD.

https://fedoraproject.org/wiki/Licensing:BSD?rd=Licensing/BSD#BSDwithAdvertising

File: rpm/libpcap.spec (hunk 1 above)
Comment (mkosola) [reply rpm/libpcap.spec]: It is done already on commit e137e820397da2554cfaef60391685feba432d53

File: rpm/libpcap.spec (hunk 2 above)
Comment (mkosola) [reply rpm/libpcap.spec]: It is done already on commit e137e820397da2554cfaef60391685feba432d53

File: rpm/libpcap.spec (hunk 3 above)
Comment (mkosola) [reply rpm/libpcap.spec]: Yeah, maybe I will change it to License: BSD

**Comment by saukko** [comment]
👍

**Pull request merged**